docs/documentation_routes.js

```diff
@@ -77,6 +77,10 @@ router.get('/make-first-prototype/add-questions', function (req, res) {
   res.redirect('/docs/make-first-prototype/use-components')
 })
 
+router.get('/docs/templates/check-your-answers', function (req, res) {
```

Comment:
I'm so sorry! you don't need `docs` either as this router is 'mounted' at `docs`

Reply:
No worries - I should pay more attention to the paths.

```suggestion
router.get('/templates/check-your-answers', function (req, res) {
```